mission_requirements:
  undocking:
    requires_camera: true
    requires_sonar: true
  pipeline_inspection:
    requires_camera: true
    requires_sonar: true
  wreckage_inspection:
    requires_camera: true
    requires_sonar: true
  return_to_dock:
    requires_camera: false
    requires_sonar: true
  docking:
    requires_camera: true
    requires_sonar: true